python/paddle/fluid/dygraph/dygraph_to_static/ast_transformer.py

```diff
@@ -240,5 +240,8 @@ def convert_to_static(dyfunc):
 
     # Get static_func from AST
     func_name = dygraph_to_static.get_module_name()
-    static_func, file_name = ast_to_func(root_wrapper.node, func_name)
+    # Transform into callable function
+    source_file = inspect.getfile(dyfunc)
+    static_func, file_name = ast_to_func(root_wrapper.node, func_name,
+                                         source_file)
```

Comment:
Can the code `source_file = inspect.getfile(dyfunc)` be moved to the function `ast_to_func, because it is one  step of ast_to_func and  `source_file` is only used in `ast_to_func`.
```
static_func, file_name = ast_to_func(root_wrapper.node, func_name,dyfunc)
```

```
def ast_to_func(...):
   source_file = inspect.getfile(dyfunc)
```

Reply:
Thanks. I refined the params of `ast_to_func` and only `dyfunc` was required.

Reply:
👍
`dygraph_to_static.get_module_name()` won't be used, we can remove it next PR.